### ISSUE TITLE
Can't run RoomRxJavaKotlin sample

### ISSUE THREAD

**Issue by undeadspez** [android/architecture-components-samples#717]
![Screenshot from 2019-09-08 16-31-10](https://user-images.githubusercontent.com/32403828/64488924-3cc3dc00-d256-11e9-9284-327f2151595d.png)

After opening RoomRxJavaKotlin sample in Android Studio (ver. 3.5 stable), can't run the app due to the following error:
```
2019-09-08 16:34:06.928 12589-12589/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.android.observability, PID: 12589
    java.lang.RuntimeException: Cannot create an instance of class com.example.android.observability.ui.UserViewModel
        at androidx.lifecycle.ViewModelProvider$NewInstanceFactory.create(ViewModelProvider.java:213)
        at androidx.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.java:270)
        at androidx.lifecycle.SavedStateViewModelFactory.create(SavedStateViewModelFactory.java:94)
        at androidx.lifecycle.AbstractSavedStateViewModelFactory.create(AbstractSavedStateViewModelFactory.java:69)
        at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.java:177)
        at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.java:145)
        at androidx.lifecycle.ViewModelLazy.getValue(ViewModelProvider.kt:54)
        at androidx.lifecycle.ViewModelLazy.getValue(ViewModelProvider.kt:41)
        at com.example.android.observability.ui.UserActivity.getViewModel(Unknown Source:7)
        at com.example.android.observability.ui.UserActivity.onStart(UserActivity.kt:54)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1391)
        at android.app.Activity.performStart(Activity.java:7165)
        at android.app.ActivityThread.handleStartActivity(ActivityThread.java:2975)
        at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:180)
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:165)
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:142)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6718)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.InstantiationException: java.lang.Class<com.example.android.observability.ui.UserViewModel> has no zero argument constructor
        at java.lang.Class.newInstance(Native Method)
        at androidx.lifecycle.ViewModelProvider$NewInstanceFactory.create(ViewModelProvider.java:211)
        at androidx.lifecycle.ViewModelProvider$AndroidViewModelFactory.create(ViewModelProvider.java:270) 
        at androidx.lifecycle.SavedStateViewModelFactory.create(SavedStateViewModelFactory.java:94) 
        at androidx.lifecycle.AbstractSavedStateViewModelFactory.create(AbstractSavedStateViewModelFactory.java:69) 
        at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.java:177) 
        at androidx.lifecycle.ViewModelProvider.get(ViewModelProvider.java:145) 
        at androidx.lifecycle.ViewModelLazy.getValue(ViewModelProvider.kt:54) 
        at androidx.lifecycle.ViewModelLazy.getValue(ViewModelProvider.kt:41) 
        at com.example.android.observability.ui.UserActivity.getViewModel(Unknown Source:7) 
        at com.example.android.observability.ui.UserActivity.onStart(UserActivity.kt:54) 
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1391) 
        at android.app.Activity.performStart(Activity.java:7165) 
        at android.app.ActivityThread.handleStartActivity(ActivityThread.java:2975) 
        at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:180) 
        at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:165) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:142) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6718) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
```
I see that `java.lang.Class<com.example.android.observability.ui.UserViewModel> has no zero argument constructor` in file [UserViewModel.kt](https://github.com/googlesamples/android-architecture-components/blob/9da853ba7ed22c53bb468609599615f88d459e9b/BasicRxJavaSampleKotlin/app/src/main/java/com/example/android/observability/ui/UserViewModel.kt), but I'm familiar neither with Kotlin nor AndroidX. How can I run the app without exceptions?
```
$ java -version
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1ubuntu1~19.04.1-b10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)
```

**Comment by giagiac** [comment]
Same problem here! Could you please give an hint. Thanks

**Comment by ianhanniballake** [comment]
This was fixed in #766
